### PR TITLE
Added mentor Rahul Tiwari 

### PR DESCRIPTION
**Pull request by DotSlashTX**
Added the following information of mentor Rahul Tiwari
-name
-github profile
-image

Image name :
Rahul_Tiw

### CODE REVIEW

**Review by isuruAb — COMMENTED** [review]
@FreakinBabaJi  please resolve conflicts

File: _data/mentors.yml [hunk 1]
```diff
@@ -92,3 +92,10 @@
 - name: Anshuman Verma
   image: ansh.jpg
   twitter: Anshumaniac12
+ 
+
+- name: Rahul Tyagi
+  github: invinciblycool
+  image: Rahul_Tiw
+
+
```
Comment (harshit98) [inline _data/mentors.yml]: Leave one line at the end.

File: _data/mentors.yml [hunk 2]
```diff
@@ -92,3 +92,10 @@
 - name: Anshuman Verma
   image: ansh.jpg
   twitter: Anshumaniac12
+ 
+
```
Comment (harshit98) [inline _data/mentors.yml]: Add only single space.

**Comment by harshit98** [comment]
@FreakinBabaJi Good work! Please resolve conflicts. Thanks!

**Comment by DotSlashTX** [comment]
Can you please explain a bit? I really suck at git :(

**Comment by akshatnitd** [comment]
Merged :rocket:

**Pull request merged**